CHANGELOG.md

```diff
@@ -127,7 +127,25 @@ binary | sha1 hash | md5 hash
 
 ## Known Issues and Important Steps before Upgrading
 
-* *Instructions coming soon*
+The following versions of Docker Engine are supported - *v1.10*, *v1.11*
```

Comment:
should we point to our validation issue for each release here, so some user can take a look at our validation results and issues if they wanted to know more detail?